xonsh/history/dummy.py

```diff
@@ -7,6 +7,9 @@
 class DummyHistory(History):
     """A dummy implement of history backend."""
 
+    def __init__(self, **meta):
+        super().__init__(meta)
```

Comment:
this should pass the '**kwargs'?

Reply:
I cribbed it from `JsonHistory.__init__`. I did forget the `**`, I'll fix it.